src/main/java/it/smartcommunitylab/aac/webauthn/WebAuthnIdentityAuthority.java

```diff
@@ -0,0 +1,305 @@
+package it.smartcommunitylab.aac.webauthn;
+
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeUnit;
+import java.util.regex.Pattern;
+import java.util.stream.Collectors;
+
+import com.google.common.cache.CacheBuilder;
+import com.google.common.cache.CacheLoader;
+import com.google.common.cache.LoadingCache;
+import com.google.common.util.concurrent.UncheckedExecutionException;
+
+import org.springframework.beans.factory.InitializingBean;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.util.Assert;
+import org.springframework.util.StringUtils;
+
+import it.smartcommunitylab.aac.SystemKeys;
+import it.smartcommunitylab.aac.common.NoSuchProviderException;
+import it.smartcommunitylab.aac.common.RegistrationException;
+import it.smartcommunitylab.aac.core.authorities.IdentityAuthority;
+import it.smartcommunitylab.aac.core.base.ConfigurableIdentityProvider;
+import it.smartcommunitylab.aac.core.provider.IdentityProvider;
+import it.smartcommunitylab.aac.core.provider.IdentityService;
+import it.smartcommunitylab.aac.core.provider.ProviderRepository;
+import it.smartcommunitylab.aac.core.service.UserEntityService;
+import it.smartcommunitylab.aac.webauthn.provider.WebAuthnIdentityProviderConfig;
+import it.smartcommunitylab.aac.webauthn.provider.WebAuthnIdentityProviderConfigMap;
+import it.smartcommunitylab.aac.webauthn.provider.WebAuthnIdentityService;
+import it.smartcommunitylab.aac.webauthn.service.WebAuthnUserAccountService;
+
+@Service
+public class WebAuthnIdentityAuthority implements IdentityAuthority, InitializingBean {
+    public static final String AUTHORITY_URL = "/auth/webauthn/";
+    private WebAuthnIdentityProviderConfigMap defaultProviderConfig;
+    private WebAuthnIdentityProviderConfig template;
```

Comment:
drop the template

Reply:
Done in a5a24ff7cbdef1771273f5d30e6eb10e09f4311c